users/views/bind_company_view.py

```diff
@@ -0,0 +1,306 @@
+                return Response(
+                    {"error": "Investor is already associated with another user."},
+                    status=status.HTTP_400_BAD_REQUEST
+                )
+
+            investor.user = user
+            investor.save()
+
+            return Response({
+                "message": f"Successfully bound to existing investor: {company_name}",
+                "company_type": "investor",
+                "company_id": investor.id
+            }, status=status.HTTP_200_OK)
+
+        except Investor.DoesNotExist:
+            investor = self._create_new_investor(user, company_name)
+
+            return Response({
+                "message": f"Successfully created and bound to new investor: {company_name}",
+                "company_type": "investor",
+                "company_id": investor.id
+            }, status=status.HTTP_201_CREATED)
+
+    def _create_default_industry_and_location(self):
+        """
+        Create or get default industry and location instances.
+
+        Returns:
+            tuple: (industry_instance, location_instance)
+        """
+        industry, _ = Industry.objects.get_or_create(
+            name="Unknown",
+            defaults={'description': 'Default unknown industry'}
+        )
+
+        location, _ = Location.objects.get_or_create(
+            city="Unknown",
+            country="Unknown",
+            defaults={'region': 'Unknown'}
+        )
+
+        return industry, location
+
+    def _create_new_startup(self, user, company_name):
+        """
+        Create a new startup instance with default values.
+
+        Args:
+            user: User to associate with the startup
+            company_name: Name of the startup
+
+        Returns:
+            Startup instance
+        """
+        industry, location = self._create_default_industry_and_location()
+
+        return Startup.objects.create(
+            user=user,
+            company_name=company_name,
+            industry=industry,
+            location=location,
+            email=user.email,
+            founded_year=datetime.datetime.now().year,
+            team_size=1,
+            stage=Stage.IDEA
+        )
+
+    def _create_new_investor(self, user, company_name):
+        """
+        Create a new investor instance with default values.
+
+        Args:
+            user: User to associate with the investor
+            company_name: Name of the investor
+
+        Returns:
+            Investor instance
+        """
+        industry, location = self._create_default_industry_and_location()
+
+        return Investor.objects.create(
+            user=user,
+            company_name=company_name,
+            industry=industry,
+            location=location,
+            email=user.email,
+            founded_year=datetime.datetime.now().year,
+            team_size=1,
+            stage=Stage.MVP,
+            fund_size=0
+        )
```

Comment:
No validation for duplicate investor company name (case-insensitive) before creation. Check for existing Investor with case-insensitive name before creating.
```
if Investor.objects.filter(company_name__iexact=company_name).exists():
    raise ValidationError("An investor with this name already exists.")
```